examples/sveltekit-server/.eslintrc.cjs

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Can we make this a .mjs file? Actually, since we have `type: "module"` (which is good), we could just use .js (unless there is a standard).